model/task/db_test.go

```diff
@@ -622,23 +622,33 @@ func TestDisplayStatus(t *testing.T) {
 			},
 		},
 	}
+	// No CheckStatuses for t12 to avoid paradox
```

Comment:
What does this mean?

Reply:
It doesn't seem like a paradox to me, so I don't really know. I just moved this comment because it was on the wrong line.